README.md

```diff
@@ -1,11 +1,28 @@
 # Load Secrets from 1Password - GitHub Action
 
-The action to load secrets from [1Password Connect](https://1password.com/secrets/) into GitHub Actions.
+This action loads secrets from 1Password into GitHub Actions using [1Password Connect](https://1password.com/secrets/) or a Service Account.
+
 
 Specify right from your workflow YAML which secrets from 1Password should be loaded into your job, and the action will make them available as environment variables for the next steps.
 
 ## Usage
 
+You can configure the action to use either 1Password Connect instance or a 1Password Service Account. Service Accounts are currently in Beta and are only available to select users.
```

Comment:
We should mention that Service Accounts are in beta. Something like this could do it:
```suggestion
You can configure the action to use either 1Password Connect instance or a 1Password Service Account (currently in beta). Service Accounts are currently in Beta and are only available to select users.
```

We can also bold it if needed.

Reply:
We mention this in the next sentence.
> Service Accounts are currently in Beta and are only available to select users.